inference/core/workflows/core_steps/flow_control/rate_limiter/v1.py

```diff
@@ -15,6 +19,8 @@
     WorkflowBlockManifest,
 )
 
+PATTERN_STR = r"(^\$inputs.[A-Za-z_0-9\-]+$)"
```

Comment:
why this pattern change is needed?

Reply:
would it work in UI? I have a feeling it would let it construct the workflow and then fail in compilation

Reply:
I'm pretty sure it does, but if it doesn't I'll follow up with a PR there to respect it.

Reply:
ok, the same discussion in the other PR - suggested solution there